src/index.html

```diff
@@ -8,7 +8,90 @@
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body class="main-page">
+    <header class="container">
+      <div class="logo">
+        <a
+          href="#home">
+          <img src="images/logo.png" alt="logo">
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="is-active nav__link"
+              href="#apple"
+            >
```

Comment:
Only when tag have more then 3 attributes it should be multiline
```suggestion
            <a class="is-active nav__link" href="#apple">
```